### DISCUSSION TITLE
What's __WAKU_SSR_ENABLED__ for?

### DISCUSSION THREAD

**Discussion by SunHuawei** [wakujs/waku#241] [Q&A]
Hey, would anybody explain what's `__WAKU_SSR_ENABLED__` for?
No doc for now. I can only see it's been set to true here https://github.com/dai-shi/waku/blob/096e02f661c0750a4a2dcadb17a4fcea8d2ceb94/packages/waku/src/lib/rsc/html-renderer.ts#L128.
Not sure if anyone can give me some hints, so that would save me days to read the source code. Thanks.

**Comment by dai-shi** [comment]
We don't have any docs about the internal implementation now. We should start working on concept docs, usage docs, and api docs. For internal implementation docs, source code will be only something you can refer in the meantime.

https://github.com/dai-shi/waku/blob/096e02f661c0750a4a2dcadb17a4fcea8d2ceb94/examples/01_counter/src/main.tsx#L13-L17

`__WAKU_SSR_ENABLED__` is used on the client side to change if it requires hydration.
